provider/resources.go

```diff
@@ -22,8 +22,10 @@ import (
 
 	"github.com/pulumi/pulumi-spotinst/provider/v3/pkg/version"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/x"
```

Comment:
i'm learning about tfbridge still, why is this subpackage called `x`

Reply:
`x` for experimental (context in slack thread [here](https://pulumi.slack.com/archives/C04HZ190SN7/p1676400994273059?thread_ts=1676399975.017349&cid=C04HZ190SN7))